src/test/java/org/folio/service/job/JobExecutionServiceUnitTest.java

```diff
@@ -363,6 +369,32 @@ void shouldPrepareJobExecutionSuccessfully_whenJobExecutionStartDateIsNull(Vertx
     });
   }
 
+  @Test
+  void expireJobExecutionsShouldSetStatusToFail(VertxTestContext context) {
+    // given
+    JobExecution jobExecution = new JobExecution()
+      .withExportedFiles(Sets.newHashSet())
```

Comment:
Can you please write a new test/or modify this test to consider the time factor as well. 
This is to make sure that we are getting back entries for the pas hour correctly

Reply:
will be a new test as it will be integration test.